src/app/layout.tsx

```diff
@@ -27,6 +27,10 @@ export default function RootLayout({
       <head>
         <title>GDI - User Portal</title>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta
+          httpEquiv="Content-Security-Policy"
+          content={`default-src ${process.env.NEXTAUTH_URL} ${process.env.API_PORTAL_URL} 'self' data: 'unsafe-inline' 'unsafe-eval'`}
```

Comment:
important: after changing the code to comply to Next.js docs, if you still need to add the api URL, I recommend creating a dedicated env variable for `CSP_TRUSTED_SOURCES` and there we add the api domain and other links, if necessary.

Reply:
Actually, let's have a fully generic environment variable for `CSP_HEADER`, as we may have other problems in our environments.